internal/services/disks/disk_pool_iscsi_target_lun_resource_test.go

```diff
@@ -220,6 +220,11 @@ resource "azurerm_disk_pool_iscsi_target_lun" "test%[2]d" {
   iscsi_target_id                      = azurerm_disk_pool_iscsi_target.test.id
   disk_pool_managed_disk_attachment_id = azurerm_disk_pool_managed_disk_attachment.test[%[2]d].id
   name                                 = "test-%[2]d"
+
+  timeouts {
+    create = "60m"
+    delete = "60m"
+  }
```

Comment:
instead of doing this should we not increase the defaults for all users rather then just fix the tests?

Reply:
Yes I think we can probably do that